src/Wallabag/CoreBundle/Controller/TagController.php

```diff
@@ -222,4 +234,14 @@ public function tagThisSearchAction($filter, Request $request)
 
         return $this->redirect($this->get('wallabag_core.helper.redirect')->to($request->headers->get('referer'), '', true));
     }
+
+    /**
+     * Check if the logged user can manage the given entry.
+     */
+    private function checkUserAction(Entry $entry)
```

Comment:
@j0k3r looks like to be a hotfix and it's understandable :)
but wouldn't a cleaner solution be the use of Symfony voters? what do you think?

Reply:
That's a good idea!

Reply:
just created #6300 to cover this